workflow/tests/base_results/results.csv

```diff
@@ -11,17 +11,17 @@ base-appliances-oil.xml,32.24,32.24,14.04,4.87,0.0,0.0,0.0,0.0,0.0,0.37,3.55,0.6
 base-appliances-wood.xml,32.24,32.24,14.04,0.0,0.0,4.87,0.0,0.0,0.0,0.37,3.55,0.69,9.16,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,0.14,0.1,0.0,2.11,8.37,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.04,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,1.8,3.07,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,13.27,11.59,9.23,0.61,0.0,0.0,0.0,0.0,2005.0,3600.0,20.43,20.75,0.0,3.89,3.98,0.54,4.05,0.71,-0.08,0.0,0.0,3.17,-0.1,5.19,0.0,0.51,0.0,3.23,-11.93,0.0,-0.05,-0.47,-0.02,3.8,-0.03,8.88,0.0,0.0,-6.02,-0.05,-1.1,-6.64,-0.1,0.0,3.07,10.33,1355.0,998.0,11400.0,2616.0
 base-atticroof-cathedral.xml,34.79,34.79,20.12,0.0,0.0,0.0,0.0,0.0,0.0,0.52,3.22,0.61,9.17,0.0,0.0,4.51,0.0,0.33,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.11,8.37,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,20.12,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,19.02,10.02,9.23,0.62,0.0,0.0,0.0,0.0,2176.0,3209.0,21.15,17.51,7.24,0.0,4.59,0.55,3.97,0.71,0.46,0.0,0.0,2.85,-0.22,9.71,0.0,0.76,0.0,0.0,-11.66,0.22,0.0,-0.43,0.0,3.96,0.0,12.49,0.0,0.0,-5.89,-0.17,-1.71,-7.86,-0.13,0.0,0.0,9.63,1355.0,998.0,11400.0,2616.0
-base-atticroof-conditioned.xml,39.6,39.6,19.55,0.0,0.0,0.0,0.0,0.0,0.0,0.51,3.88,0.77,9.07,0.0,0.0,5.75,0.0,0.4,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.11,11.17,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,19.55,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,18.49,12.9,9.18,0.61,0.0,0.0,0.0,3.0,2438.0,3580.0,24.46,21.3,4.88,1.23,5.96,0.55,4.27,0.7,0.69,0.0,0.0,3.34,-0.19,7.37,0.0,0.76,0.0,2.36,-13.53,0.06,0.04,-0.48,-0.01,3.55,-0.02,13.91,0.0,0.0,-5.8,-0.13,-1.47,-8.78,-0.15,0.0,0.92,11.39,1355.0,998.0,11400.0,2522.0
+base-atticroof-conditioned.xml,39.42,39.42,19.07,0.0,0.0,0.0,0.0,0.0,0.0,0.5,3.74,0.74,9.07,0.0,0.0,5.75,0.0,0.4,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.32,0.37,1.51,1.53,0.0,2.11,11.17,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,19.07,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,18.03,12.31,9.18,0.61,0.0,0.0,0.0,0.0,2436.0,3580.0,23.84,20.99,4.86,1.23,6.13,0.54,4.25,0.7,0.23,0.0,0.0,3.32,-0.17,7.35,0.0,0.76,0.0,2.3,-13.54,0.06,0.04,-0.49,-0.01,3.54,-0.01,12.9,0.0,0.0,-5.79,-0.12,-1.48,-8.3,-0.15,0.0,0.88,11.38,1355.0,998.0,11400.0,2522.0
```

Comment:
This file is changing more than it should too. Seems to be related to a change in window area.